resolwe_bio/management/commands/generate_samples.py

```diff
@@ -67,15 +70,22 @@ def generate_expressions(self, gene_ids, path):
             js_out = '{{"exp_json":{}}}'.format(json.dumps({'genes': genes}, separators=(',', ':')))
             json_file.write(js_out)
 
-    def generate_reads_descriptor(self):
+    @staticmethod
+    def generate_reads_descriptor(annotated):
         barcodes = ['ATGCATGC', 'TACGTACG']
         instruments = ['Illumina HiSeq X', 'Illumina HiSeq 3000', 'Illumina HiSeq 3000']
         descriptor = {'barcode': random.choice(barcodes),
                       'instrument_type': random.choice(instruments),
                       'barcode_removed': True}
+
+        if annotated:
```

Comment:
`seq_date` should be set for samples and presamples. In reality it is set by the script that uploads the files from the sequencer, this is the date of sequencing.